app/src/sider/3-soknadstilpasser/felles/personalia/Personalia.tsx

```diff
@@ -15,24 +17,35 @@ import FlerePersonerPanel from "./paneler/FlerePersoner";
 import TiltaksBedriftPanel from "./paneler/TiltaksBedrift";
 import NesteKnapp from "./knapper/Neste";
 
+interface ValgtSoknad {
+  valgtSoknadsobjekt: Soknadsobjekt;
+  klageSoknadsobjekt: Soknadsobjekt;
+}
+
 interface Routes {
   personEllerBedrift: string;
 }
 
-interface Props {
-  personaliaKontekst: PersonaliaKontekst;
-}
+type MergedProps = ValgtSoknad &
+  Personalia &
+  RouteComponentProps<Routes> &
+  InjectedIntlProps;
 
-type MergedProps = Props & RouteComponentProps<Routes> & InjectedIntlProps;
-class Personalia extends React.Component<MergedProps> {
+class VisPersonalia extends Component<MergedProps> {
   handleSubmit = (e: any) => {
-    const { history, personaliaKontekst, match } = this.props;
-    personaliaKontekst.resetState();
-    if (erGyldigFodselsnummer(e.fodselsnummer)) {
+    const { history, match } = this.props;
+    this.props.resetState();
+    if (
+      e.fodselsnummer.fodselsnummer &&
+      erGyldigFodselsnummer(e.fodselsnummer.fodselsnummer)
+    ) {
```

Comment:
Burde vi hatt noe validering på valgt enhet her? Dersom man blir presentert for muligheten til å velge enhet må man gjøre det før man går videre, samme på neste alternativ